packages/volto/src/helpers/Url/Url.js

```diff
@@ -139,7 +139,10 @@ export const isCmsUi = memoize((currentPathname) => {
   // because the regexp test does not take that into account
   // https://github.com/plone/volto/issues/870
   return settings.nonContentRoutes.reduce(
-    (acc, route) => acc || new RegExp(route).test(`/${fullPath}`),
+    (acc, route) =>
+      acc ||
+      (settings.publicUiRoutes.indexOf(route) < 0 &&
+        new RegExp(route).test(`/${fullPath}`)),
```

Comment:
```suggestion
        new RegExp(route).test(fullPath)),
```
wen need the additional `/` ?

Reply:
yes, fullPath isn't an entry of the list above, but the current path 
nothing changed from before. it was as is and worked well

Reply:
Does `currentPathname` include or exclude a leading forward slash (`/`)?

If it includes a slash, then we don't need to use one in the regexp test.

If it excludes a slash, then we need a slash in the regexp test.